store/orm/orm.go

```diff
@@ -214,17 +232,30 @@ func (orm *ORM) convenientTransaction(callback func(*gorm.DB) error) error {
 // SaveJobRun updates UpdatedAt for a JobRun and saves it
 func (orm *ORM) SaveJobRun(run *models.JobRun) error {
 	return orm.convenientTransaction(func(dbtx *gorm.DB) error {
-		return dbtx.Save(run).Error
+		return dbtx.Unscoped().Omit("deleted_at").Save(run).Error
```

Comment:
Why do we need the `Omit`? Possibly overwrites a previously set value?

Reply:
This is related to the following scenario:

> After archiving a job, a run already in progress must become and continue to be archived. Future saves of run should not flip back the deleted at and unarchive.

If we have a run already in progress, and a user invokes the CLI command to asynchronously archive all jobs and the run, we don't want a subsequent save to flip back deleted_at to null because that's what's held in memory.